packages/lib-sourcify/test/functions.spec.ts

```diff
@@ -18,17 +18,20 @@ import { Metadata, MissingSources } from '../src/lib/types';
 
 describe('Verify Solidity Compiler', () => {
   it('Should fetch latest SolcJS compiler', async () => {
-    await getSolcJs();
+    expect(await getSolcJs()).not.equals(null);
   });
   it('Should fetch SolcJS compiler passing only version', async () => {
-    await getSolcJs('0.8.17+commit.8df45f5f');
-  });
-  it('Should fetch latest solc from github', async () => {
-    await getSolcExecutable('linux-amd64', '0.8.9+commit.e5eed63a');
+    expect(await getSolcJs('0.8.17+commit.8df45f5f')).not.equals(false);
   });
+  if (process.platform === 'linux') {
+    it('Should fetch latest solc from github', async () => {
+      expect(
+        await getSolcExecutable('linux-amd64', '0.8.9+commit.e5eed63a')
+      ).not.equals(null);
+    });
+  }
```

Comment:
Let's complete it with the Mac and Windows tests

Reply:
at the end the tests are needed for circleci right? are macx64 and winx64 needed?

Reply:
Mostly yes. This will be implicit in the missing test mentioned above anyway, so fine this way.

Reply:
But that is already tested by other tests, that's why is missing here, if you check the coverage that branch is already covered like x15 times